src/pageObject.js

```diff
@@ -1,16 +1,42 @@
-'use strict';
+"use strict";
 
 class Header {
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
-};
-
-class PageObject {
+class PageObject extends Header {
+  constructor(url) {
+    super();
+    this.url = url;
+    this.header = new Header();
```

Comment:
You don't need to create a new instance of `Header` inside `PageObject` since `PageObject` already extends `Header`. This means that `PageObject` already has access to `Header`'s methods and properties. Remove this line to avoid unnecessary object creation.